gcloud-java-resourcemanager/src/main/java/com/google/gcloud/spi/ResourceManagerRpc.java

```diff
@@ -110,7 +111,7 @@ public int pageSize() {
 
   Policy getIamPolicy(String projectId) throws ResourceManagerException;
 
-  boolean replaceIamPolicy(String projectId, Policy policy) throws ResourceManagerException;
+  Policy replaceIamPolicy(String projectId, Policy policy) throws ResourceManagerException;
```

Comment:
I'll take the policy related methods and Permission enum out.  I was originally going to keep the policy methods in case the policy manager doesn't get included right away.  Thinking about the timeline some more, that's probably not going to be an issue.